tests/unit/amo/components/TestSearchResult.js

```diff
@@ -131,19 +131,197 @@ describe(__filename, () => {
   });
 
   it('adds a theme-specific class', () => {
-    const addon = createInternalAddon({
-      ...fakeAddon,
-      type: ADDON_TYPE_THEME,
-      theme_data: {
-        previewURL:
-          'https://addons.cdn.mozilla.net/user-media/addons/334902/preview_large.jpg?1313374873',
-      },
+    const root = render({
+      addon: createInternalAddon({
+        ...fakeAddon,
+        type: ADDON_TYPE_THEME,
+        theme_data: {
+          previewURL:
+            'https://addons.cdn.mozilla.net/user-media/addons/334902/preview_large.jpg?1313374873',
+        },
+      }),
     });
-    const root = render({ addon });
 
     expect(root).toHaveClassName('SearchResult--theme');
   });
 
+  it('does not render an image if the isAllowedOrigin is false', () => {
+    const root = render({
+      addon: createInternalAddon({
+        ...fakeAddon,
+        type: ADDON_TYPE_STATIC_THEME,
+        previews: [
+          {
+            ...fakePreview,
+            thumbnail_url: 'http://example.com',
+          },
+        ],
+      }),
+    });
+
+    expect(root.find('.SearchResult-icon')).toHaveLength(0);
+  });
+
+  it("renders an image's alt attribute as its addon name", () => {
+    const alt = 'pretty image';
+    const root = render({
+      addon: createInternalAddon({
+        ...fakeAddon,
+        name: alt,
+        type: ADDON_TYPE_THEME,
+        previews: [
```

Comment:
Because we're only testing the `img`'s `alt` property, we shouldn't need to override the `previews` property at all.